Gems/Atom/Feature/Common/Assets/Materials/Pipelines/MainPipeline/ForwardPass_StandardLighting.shader.template

```diff
@@ -0,0 +1,43 @@
+{
+    "Source" : "INSERT_AZSL_PATH_HERE",
+
+    "DepthStencilState" :
+    {
+        "Depth" :
+        {
+            "Enable" : true,
+            "CompareFunc" : "GreaterEqual"
+        },
+        "Stencil" :
+        {
+            "Enable" : true,
```

Comment:
Yes they do, and that's actually fine since this shader template is defined inside the material pipeline. Some other pipeline would have a different shader template and could therefore have different stencil settings.

Reply:
Something that would be interesting to consider is whether the material type could have some influence over the shader's depth/stencil settings. I think in that case we would use the material pipeline script concept, possibly expanding its API if needed. I think we can address that when the use case arises, though.